netmanager/src/views/components/DataDisplay/DeviceView/DeviceDeployStatus.js

```diff
@@ -332,27 +355,49 @@ export default function DeviceDeployStatus({ deviceData }) {
     }
   };
 
+  const createSiteOptions = () => {
+    const options = [];
+    sites.map((site) =>
+      options.push({
+        value: site._id,
+        label: site.name || site.description || site.generated_name,
```

Comment:
lovely!

Reply:
Thia

> This is beyond lovely! @ja-odur
> For a good user experience, please add a back navigation on the site registry page accordingly. From user (Deo Okure) experience on device registry page, this is actually much needed so I will go ahead and request this change. :)
> 
> See screenshot
> <img alt="Screenshot 2021-06-24 at 20 40 44" width="1637" src="https://user-images.githubusercontent.com/1590213/123308873-e445f780-d52c-11eb-9dce-ea265677e226.png">

This has been resolved @Baalmart